app/i18n/locales/zh-Hant.json

```diff
@@ -200,18 +200,19 @@
   "transfer.summary.recoveredBalance.label": "恢復的餘額",
   "transfer.summary.transactionFee.label": "交易費用",
   "transfer.summary.transferButton.label": "轉移資金",
-  "wallet.add.dialog.create.description": "Create wallet",
```

Comment:
please don't change non-EN files 🙏

Reply:
Yeah, going to undo all those files, and from now on will never change non-EN.